sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonSensorTest.java

```diff
@@ -210,7 +209,9 @@ public boolean scanWithoutParsing(PythonInputFileContext inputFile) {
   private final AnalysisWarningsWrapper analysisWarning = mock(AnalysisWarningsWrapper.class);
 
   @org.junit.Rule
-  public LogTester logTester = new LogTester();
+  public LogTester logTester = new LogTester().setLevel(Level.DEBUG);
+  @org.junit.Rule
+  public LogTester traceLogTester = new LogTester().setLevel(Level.TRACE);
```

Comment:
Do we need both LogTester? Is it possible to make it work with the `logTester` set to TRACE level?

Reply:
it could be but then it requires assertion changes